### ISSUE TITLE
Display emeritus maintainers on https://eng.istio.io/

### ISSUE THREAD

**Issue by howardjohn** [istio/community#976]
https://eng.istio.io/ has a list of all maintainers, but not emeritus ones. We received feedback that it would be great to list emeritus maintainers there as well

cc @therealmitchconnors

**Comment by ericvn** [comment]
I do believe there is code in itio/bots to handle the display of this. I'm just not sure how to set the flag in the database.

**Comment by therealmitchconnors** [comment]
Yes, this request has come up in the past.  This would likely require a googler who can dig into the spanner db, tweak the bots repo, and redeploy the code into the bots cluster.  

In the meantime, we do have a yaml file tracking emeritus maintainers in the community repo.
